app/src/main/java/com/pajato/android/gamechat/common/PlayLocationManager.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2016 Pajato Technologies, Inc.
+ *
+ * This file is part of Pajato GameChat.
+
+ * GameChat is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * GameChat is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License along with GameChat.  If not,
+ * see <http://www.gnu.org/licenses/>.
+ */
+package com.pajato.android.gamechat.common;
+
+import android.app.Activity;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.PopupWindow;
+
+import com.pajato.android.gamechat.R;
+import com.pajato.android.gamechat.chat.model.Room;
+import com.pajato.android.gamechat.common.adapter.ListItem;
+import com.pajato.android.gamechat.common.adapter.PlayLocationMenuAdapter;
+import com.pajato.android.gamechat.common.adapter.PlayLocationMenuEntry;
+import com.pajato.android.gamechat.common.model.Account;
+import com.pajato.android.gamechat.database.AccountManager;
+import com.pajato.android.gamechat.database.ExperienceManager;
+import com.pajato.android.gamechat.database.RoomManager;
+import com.pajato.android.gamechat.exp.BaseExperienceFragment;
+import com.pajato.android.gamechat.exp.Experience;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static android.widget.LinearLayout.VERTICAL;
+
+public enum PlayLocationManager {
+/**
```

Comment:
This doc comment should precede the public enum declaration, I do believe.  It would be reasonable to augment the doc comment with a code comment following the enum declaration if you so chose.